frontend/src/components/pages/RegistrantExperience/SessionSelection/SessionCard.tsx

```diff
@@ -0,0 +1,150 @@
+import React from "react";
+import { Box, Text, VStack, Flex, useMediaQuery } from "@chakra-ui/react";
+import { CampSessionResponse } from "../../../../types/CampsTypes";
+import { SessionCardState } from "./SessionSelectionTypes";
+import {
+  sessionCardBoldTextStyles,
+  sessionCardDatesTextStyles,
+  sessionCardDetailsTextStyles,
+} from "./SessionSelectionStyles";
+import { adjustTimeToAmPm } from "../../../../utils/CampUtils";
+
+const WAITLISTED_COLOR = "red.100";
+const AVAILABLE_COLOR = "green.100";
+
+type SessionCardDetailsProps = {
+  fee: number;
+  startTime: string;
+  endTime: string;
+  campSession: CampSessionResponse;
+};
+
+const SessionCardDetails = ({
+  fee,
+  startTime,
+  endTime,
+  campSession,
+}: SessionCardDetailsProps): React.ReactElement => {
+  const getCampSessionDates = () => {
+    const startDate = new Date(campSession.dates[0]);
+    const endDate = new Date(campSession.dates[campSession.dates.length - 1]);
+    return `${startDate.toLocaleString("default", {
+      month: "short",
+    })} ${startDate.getDay()} - ${endDate.toLocaleString("default", {
+      month: "short",
+    })} ${endDate.getDay()}, ${endDate.getFullYear()}`;
+  };
+
+  const formattedTimeString = (start: string, end: string): string => {
+    return `${adjustTimeToAmPm(start)} - ${adjustTimeToAmPm(end)}`;
+  };
+
+  const [isMobile] = useMediaQuery("(max-width: 767px)");
```

Comment:
Is this media query accurately determining if the screen is mobile or not? It seems like it's determining if the left side of the UI is less than 767px since I am seeing the `time . fees` on desktop and the fees in a new line on mobile:

<img width="660" alt="image" src="https://user-images.githubusercontent.com/57159970/209547188-fdfa3f79-d7e6-4720-aa54-1f8f3cc7da82.png">
Desktop ^^ 

<img width="424" alt="image" src="https://user-images.githubusercontent.com/57159970/209547216-cda5d1e2-3ebf-4951-a03a-3118e65c6da5.png">
mobile ^^

Reply:
I don't think this is a problem but I just wanted to bring it up

Reply:
I think I just call it "mobile" as a differentiation between the phone/tablet layout vs. the desktop layout -- it's not a strict constraint, you can make the desktop browser into phone dimensions too